### DISCUSSION TITLE
Ride Ons

### DISCUSSION THREAD

**Discussion by dgcasey0325** [SauceLLC/sauce4zwift#83] [Q&A]
Anyone seeing Ride Ons not working? ROR and RON stopped working in Zwift. ROG is still working fine. I guess I should ask, is the Zwift HUD supposed to be on for it to work? I have it off because I have S4Z showing everything I need.
![Image1](https://github.com/SauceLLC/sauce4zwift/assets/38332950/bc3f727d-d339-406b-af3a-735a4ec321ad)

**Comment by mayfield** [comment]
Hi @dgcasey0325, Did you figure this out?  As far as I know rideons are working as expected but perhaps it's an issue with the Mod you're using?

**Reply by dgcasey0325** [reply]
But do you have the HUD turned off? I think it needs to be turned on to display all three Ride On  metrics.

**Comment by mayfield** [comment]
@dgcasey0325 the HUD state shouldn't affect it with the exception of HUD-less events which disable the data stream.  Sounds like you're still having issues, so can you tell me more about your setup used for the ride-ons display?  For example, is this a Mod or a "Watching" window, etc?
